### DISCUSSION TITLE
State

### DISCUSSION THREAD

**Discussion by xeruf** [alexpdp7/frtcuop#13] [General]
Saw your reddit post, have you abandoned this upon learning about Snikket, or are you using the deployment personally?
What advantages does it have over existing solutions?

**Comment by alexpdp7** [comment]
I've actually abandoned this.

My personal purpose for doing this was "unified communications, including SIP hardware phones". I think that's out of my reach with the current ecosystem. The projects trying to interop SIP with Matrix don't seem active, and XMPP/SIP interop seems to be possible (see https://cheogram.com/ ), but out of my league. Besides XMPP and Matrix, I'm not aware of many other good IM systems that work well on smartphones.

If you don't need to unify SIP hardware phones with a general IM system, I've found out that FreeSWITCH is really simple to set up, and it has covered my SIP hardware phones need reasonably.

For instant messaging, I'm resigned to having to be on Telegram and WhatsApp, because I have too many contacts there that will not switch to anything. Telegram bridging is doable, but WhatsApp bridging is too risky IMHO, so I'm giving up on that.

(For completeness, I'm investigating IRC recently too :).

What do you want do? If you want to run your own IM service, I think Snikket is likely the way to go.

**Comment by xeruf** [comment]
Yep, set up an XMPP server for friends and family, but I have whole communities that I could envision on there.
One of my dreams is to make websites (forums and messaging systems on sites) XMPP-compliant, so that people get an actual value from using XMPP: They don't need accounts, notifications and tabs of hundreds of sites, but can have it in one app. One can dream...

**Comment by alexpdp7** [comment]
(Moved this to a discussion, because I think that's more adequate than an issue).

Have you looked at Movim? It's not exactly what you're talking about, but instead of using traditional forum software, forums should instead be Movim sites. This lets people get started with a familiar web interface, and maybe later upgrade to regular XMPP clients (that also allow them to join conversations on different Movim sites).

Many Matrix web clients are basically good for this. People are getting used nowadays to Slack-like (or Discord-like) discussion (which I think is inferior for some purposes to forums/mailing lists, but oh well!)... and Matrix/Movim can do that nicely.

For "slow discussion" I still think email is the way to go.
